integration_tests/run_e2e_tests.py

```diff
@@ -387,7 +397,8 @@ def main(target, e2e_type, generate_data, clear_tests):
             "seasonal_volume",
             "table",
             "column",
-            "directional_anomalies"
```

Comment:
There was a missing comma here, so the tests for `directional_anomalies` (and `schema`) weren't running before.